.github/actions/frontend/runtime/e2e_prepare/test/action_test.yml

```diff
@@ -0,0 +1,26 @@
+name: Tests
+
+on: [push]
+
+jobs:
+  e2e_prepare:
+    runs-on: ubuntu-latest
+    steps:
+      # https://github.com/nektos/act#module_not_found
+      - uses: actions/checkout@v3
+        with:
+          path: e2e_prepare
+      - id: e2e_prepare
+        uses: ./e2e_prepare
+      - name: results_artemis_account_name
+        run: echo ${{ steps.e2e_prepare.outputs.artemis_account_name }}
+        shell: bash
+      - name: results_artemis_account_subdomain
+        run: echo ${{ steps.e2e_prepare.outputs.artemis_account_subdomain }}
+        shell: bash
+      - name: results_artemis_account_id
+        run: echo ${{ steps.e2e_prepare.outputs.artemis_account_id }}
+        shell: bash
+      - name: results_artemis_users
+        run: echo ${{ steps.e2e_prepare.outputs.artemis_users }}
+        shell: bash
```

Comment:
are these echo step required to make the test work?  mainly just trying to get my head wrapped around how the code work with this question.

Reply:
the wrapper workflow to call the action makes sense.  It appears to me that we're validating the outputs twice.  Once in action.test.ts via expect statements, and again on lines 15-26 via echo statements.